### DISCUSSION TITLE
What is an SAD disorder in psychology

### DISCUSSION THREAD

**Discussion by toshikijp** [andatoshiki/toshiki-discuss#31] [Q&A] [answered]
As the question describes

**Comment by andatoshiki** [comment, marked as answer]
# Overview - Seasonal affective disorder (SAD)

**Seasonal affective disorder (SAD) is a type of depression that comes and goes in a seasonal pattern.**

SAD is sometimes known as "winter [[depression](https://www.nhs.uk/mental-health/conditions/clinical-depression/overview/)](https://www.nhs.uk/mental-health/conditions/clinical-depression/overview/)" because the symptoms are usually more apparent and more severe during the winter.

A few people with SAD may have symptoms during the summer and feel better during the winter.

## Symptoms of SAD

Symptoms of SAD can include:

- a persistent low mood
- a loss of pleasure or interest in normal everyday activities
- irritability
- feelings of despair, guilt and worthlessness
- feeling lethargic (lacking in energy) and sleepy during the day
- sleeping for longer than normal and finding it hard to get up in the morning
- craving [[carbohydrates](https://www.nhs.uk/live-well/eat-well/food-types/starchy-foods-and-carbohydrates/)](https://www.nhs.uk/live-well/eat-well/food-types/starchy-foods-and-carbohydrates/) and gaining weight

For some people, these symptoms can be severe and have a significant impact on their day-to-day activities.

[[Read more about the symptoms of SAD](https://www.nhs.uk/mental-health/conditions/seasonal-affective-disorder-sad/symptoms/)](https://www.nhs.uk/mental-health/conditions/seasonal-affective-disorder-sad/symptoms/)

## When to see a GP

You should consider seeing a GP if you think you might have SAD and you're struggling to cope.

The GP can carry out an assessment to check your mental health. They may ask you about your mood, lifestyle, eating habits and sleeping patterns, plus any seasonal changes in your thoughts and behaviour.

[[Read more about diagnosing SAD](https://www.nhs.uk/mental-health/conditions/seasonal-affective-disorder-sad/diagnosis/)](https://www.nhs.uk/mental-health/conditions/seasonal-affective-disorder-sad/diagnosis/)

## What causes SAD?

The exact cause of SAD is not fully understood, but it's often linked to reduced exposure to sunlight during the shorter autumn and winter days.

The main theory is that a lack of sunlight might stop a part of the brain called the hypothalamus working properly, which may affect the:

- **production of melatonin** – melatonin is a hormone that makes you feel sleepy; in people with SAD, the body may produce it in higher than normal levels
- **production of serotonin** – serotonin is a hormone that affects your mood, appetite and sleep; a lack of sunlight may lead to lower serotonin levels, which is linked to feelings of depression
- **body's internal clock (circadian rhythm)** – your body uses sunlight to time various important functions, such as when you wake up, so lower light levels during the winter may disrupt your body clock and lead to symptoms of SAD

It's also possible that some people are more vulnerable to SAD as a result of their genes, as some cases appear to run in families.

## Treatments for SAD

A range of treatments are available for SAD. A GP will recommend the most suitable treatment programme for you.

The main treatments are:

- **lifestyle measures** – including getting as much natural sunlight as possible, exercising regularly and managing your stress levels
- **light therapy** – where a special lamp called a light box is used to simulate exposure to sunlight
- **talking therapies** – such as [[cognitive behavioural therapy (CBT)](https://www.nhs.uk/mental-health/talking-therapies-medicine-treatments/talking-therapies-and-counselling/cognitive-behavioural-therapy-cbt/overview/)](https://www.nhs.uk/mental-health/talking-therapies-medicine-treatments/talking-therapies-and-counselling/cognitive-behavioural-therapy-cbt/overview/) or [[counselling](https://www.nhs.uk/mental-health/talking-therapies-medicine-treatments/talking-therapies-and-counselling/counselling/)](https://www.nhs.uk/mental-health/talking-therapies-medicine-treatments/talking-therapies-and-counselling/counselling/)
- [**[antidepressant medicine](https://www.nhs.uk/mental-health/talking-therapies-medicine-treatments/medicines-and-psychiatry/antidepressants/overview/)**](https://www.nhs.uk/mental-health/talking-therapies-medicine-treatments/medicines-and-psychiatry/antidepressants/overview/) – such as [[selective serotonin reuptake inhibitors (SSRIs)](https://www.nhs.uk/mental-health/talking-therapies-medicine-treatments/medicines-and-psychiatry/ssri-antidepressants/overview/)](https://www.nhs.uk/mental-health/talking-therapies-medicine-treatments/medicines-and-psychiatry/ssri-antidepressants/overview/)
